docs/02.API-REFERENCE.md

```diff
@@ -6571,21 +6661,21 @@ so the user can release the buffer which was provided.
 
 ```c
 jerry_value_t
-jerry_create_shared_arraybuffer_external (const jerry_length_t size
+jerry_create_shared_arraybuffer_external (const jerry_length_t size,
                                           uint8_t *buffer_p,
-                                          jerry_value_free_callback_t free_cb);
+                                          void *user_p)
 ```
 
 - `size` - size of the buffer to use **in bytes** (should not be 0)
 - `buffer_p` - the buffer used for the Shared Array Buffer object (should not be a null pointer)
-- `free_cb` - the callback function called when the object is released
+- `user_p` - user pointer passed to the callback when the arraybuffer is freed
 - return value
     - the new SharedArrayBuffer as a `jerry_value_t`
     - if the `size` is zero or `buffer_p` is a null pointer this will return an empty SharedArrayBuffer.
 
 *New in version [[NEXT_RELEASE]]*.
 
-*Changed in version [[NEXT_RELEASE]]*: type of `free_cb` has been changed.
+*Changed in version [[NEXT_RELEASE]]*: `free_cb` is replaced by `user_p`.
```

Comment:
As this is already a new function (which is in any release yet), there is no need for both the "new in version  [[next_release]]" and "changed in [[new_release]]" entries. Just keep the "new in version..." one.